src/components/AccordionPanel/AccordionPanelTrigger.tsx

```diff
@@ -46,8 +46,8 @@ const accordionPanelTrigger = tv({
       'hover:shr-shadow-none',
       'focus-visible:shr-focusIndicator',
     ],
-    leftIcon: 'group-aria-expanded:shr-rotate-90',
-    rightIcon: 'group-aria-expanded:-shr-rotate-180',
+    leftIcon: 'group-aria-expanded:shr-rotate-90:shrink-0',
+    rightIcon: 'group-aria-expanded:-shr-rotate-180:shrink-0',
```

Comment:
元の記述が効かなくなってました!（Chromatic の差分はここでした）

```suggestion
    leftIcon: 'group-aria-expanded:shr-rotate-90 group-aria-expanded:shrink-0',
    rightIcon: 'group-aria-expanded:-shr-rotate-180 group-aria-expanded:shrink-0',
```

Reply:
失礼しました！今一生懸命 VRT 結果を眺めてるところでした 🙏